packages/shared/src/components/Feed.tsx

```diff
@@ -53,7 +53,10 @@ import { acquisitionKey } from './cards/AcquisitionFormCard';
 import { MarketingCtaVariant } from './cards/MarketingCta/common';
 
 export interface FeedProps<T>
-  extends Pick<UseFeedOptionalParams<T>, 'options'>,
+  extends Pick<
+      UseFeedOptionalParams<T>,
+      'options' | 'showPublicSquadsEligibility'
```

Comment:
Wouldn't this be one of the options better?

Reply:
I tried that at first, but it was a type from `useQuery` itself. `options?: UseInfiniteQueryOptions<FeedData>`

Thought it would be simpler just to add it.